lib/src/Uncertainty/Distribution/KernelMixture.cxx

```diff
@@ -661,6 +661,64 @@ KernelMixture::PointWithDescriptionCollection KernelMixture::getParametersCollec
   return parameters;
 } // getParametersCollection
 
+
+Point KernelMixture::getParameter() const
+{
+  const UnsignedInteger size = sample_.getSize();
+  Point parameter;
+  for (UnsignedInteger i = 0; i < size; ++ i)
+  {
+    parameter.add(sample_[i]);
+  }
+  parameter.add(bandwidth_);
+  return parameter;
+}
+
+Description KernelMixture::getParameterDescription() const
+{
+  const UnsignedInteger dimension = getDimension();
+  const UnsignedInteger size = sample_.getSize();
+  Description description;
+  for (UnsignedInteger i = 0; i < size; ++ i)
+  {
+    if (dimension > 1)
+      for (UnsignedInteger j = 0; j < dimension; ++ j)
+        description.add(OSS() << "x_" << i << "^" << j);
+    else
+      description.add(OSS() << "x_" << i);
+  }
+
+  if (dimension > 1)
+    for (UnsignedInteger j = 0; j < dimension; ++ j)
+      description.add(OSS() << "h_" << j);
+  else
+    description.add(OSS() << "h");
```

Comment:
Could be simplified to `description.add("h")`, but not a big deal.

Reply:
ok, done